src/blocks/modal.css

```diff
@@ -59,6 +59,21 @@
   margin-bottom: 54px;
 }
 
```

Comment:
* Some of my remarks haven't been corrected:
     1. this one https://github.com/TrainerSmooth/se_project_aroundtheus/pull/1#discussion_r1914516562 
     2. https://github.com/TrainerSmooth/se_project_aroundtheus/pull/1#discussion_r1914518477
     3. https://github.com/TrainerSmooth/se_project_aroundtheus/pull/1#discussion_r1914520284
     4. https://github.com/TrainerSmooth/se_project_aroundtheus/pull/1#discussion_r1914521104
     5. https://github.com/TrainerSmooth/se_project_aroundtheus/pull/1#discussion_r1914522355